raiden/connection_manager.py

```diff
@@ -289,3 +320,23 @@ def open_channels(self):
             self.api.get_channel_list(token_address=self.token_address)
             if channel.isopen
         ]
+
+    @property
+    def min_settle_blocks(self):
+        """Returns the minimum necessary waiting time to settle all channels.
+        """
+        channels = self.api.get_channel_list(
+            token_address=self.token_address
+        )
+        timeouts = [0]
```

Comment:
this is on purpose: I return `max(timeouts)` and want the minimum to be `0`, whatever weird values _may_ come in through the calculations in between.